.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish docs to Wiki
+
+on:
+  push:
+    paths:
+      - docs/**
+    branches:
+      - main
+      
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }}
+  USER_NAME: team-nimblehq
+  USER_EMAIL: dev@nimblehq.co
+  OWNER: ${{ github.event.repository.owner.name }}
+  REPOSITORY_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  Publish-Docs-To-Wiki:
+    runs-on: ubuntu-latest
+    steps:  
+    - uses: actions/checkout@v2
+    - name: Prepare
+      run: |
+         mkdir tmp_wiki
```

Comment:
Do we need to remove `tmp_wiki` once completed or it will do automatically? 🤔

Reply:
it's a docker container env, so it will be gone if you don't keep it (git add + git commit)